lib/views/general/_frontpage_newsletter_signup.erb

```diff
@@ -0,0 +1,6 @@
+<div class="newsletter-signup" id="newsletter-signup">
+  <img class="newsletter-signup__image" src="<%= image_path('logo-mysociety-black') %>" alt="mySociety logo" width="250px" height="53px"/>
+  <h2 class="newsletter-signup__title"><%= _("Keep up with what we're doing") %></h2>
+  <p class="newsletter-signup__subtitle">News on Transparency, Democracy, Community Climate and more…</p>
```

Comment:
Missing comma after "Community"

Reply:
I think we should add a _tiny_ bit of explanation of mySociety – a one liner.

https://www.whatdotheyknow.com/help/credits has:

> WhatDoTheyKnow is a project of mySociety, a UK organisation. mySociety builds websites that empower citizens to hold authorities to account.

"hold to account" is a bit old language now, I think.

https://www.mysociety.org/about has:

> mySociety is a not for profit group pioneering the use of online technologies to empower citizens to take their first steps towards greater civic participation.

> We help people be active citizens with technology, research and data that individuals, journalists, and civil society can use, openly and for free.

We could condense that down to something like:

> WhatDoTheyKnow is a project of mySociety, a not for profit that creates technology, research and data to empower citizens to take their first steps towards greater civic participation.

<img width="993" alt="Screenshot 2023-02-24 at 12 14 21" src="https://user-images.githubusercontent.com/282788/221176611-5926da85-5ea1-4a76-950c-19a8bea7c394.png">

@MyfanwyNixon how's this from a phrasing POV?

Reply:
Thanks @garethrees glad you like it =)
I think we should try to condense the first and second paragraph, because it feels a bit disjoint in terms of topics:

- First we have a mySociety logo [mySociety]
- Second we have a title that hints about a newsletter [Newsletter]
- But then we talk about mySociety [mySociety]
- Finally we are back at the newsletter again. [Newsletter]

So it feels a bit disjoint. I'm sure @MyfanwyNixon can give us some her magic =)

I changed her the button so it's more self explanatory.

Reply:
Yeah, agree the phrasing being a bit jumbled. I think it's roughly the right information to convey, just needs some finesse.